mindmeld/converter/README.md

```diff
@@ -0,0 +1,20 @@
+### Mindmeld Conversion Tool
+
+__Introduction__  
+This tool is designed to help make the migration from other conversational AI platforms to MindMeld as seamless as possible. MindMeld has many advantages that make it an ideal solution for building conversational assistants, and with this tool you can easily transfer your existing projects into MindMeld to expand their capabilities. Currently, we offer support for projects built in Rasa and Dialogflow.
+
+__How does this tool work?__  
+This tool will generate the basic framework of a MindMeld project, and will use the entities and intents of the existing project to populate the entities folders and create training files respectively. Dialogue state handlers will be created for each intent to handle any necessary logic behind responses, and a default configurations file will also be created, which may be changed as needed. If you are unfamiliar with the structure of MindMeld projects, code examples can be found in our MindMeld Blueprints [here](https://github.com/CiscoDevNet/mindmeld-blueprints/tree/develop/blueprints), and overviews can be found [here](https://www.mindmeld.com/docs/blueprints/overview.html).
+
+__Usage__  
+*Rasa Users:*  
+In a python script or from the command line, run the following commands
+
```

Comment:
We should also add some limitations to the current converters here. On the top of my head, these include:
1. Multiple intents being detected simultaneously
2. Language support

Reply:
I added a section for limitations, and got the list of limitations from dialogflow.py and rasa.py. Would it be better to have the list in one place on the readme (and not under the convert_project functions in the other files)?